examples/nlg_server/nlg_server.py

```diff
@@ -48,14 +48,14 @@ async def generate_response(nlg_call, domain):
     Generates the responses from the bot's domain file.
     """
     kwargs = nlg_call.get("arguments", {})
-    template = nlg_call.get("template")
+    response = nlg_call.get("response")
```

Comment:
I think this would be worth to add to the changelog. Can you please also update the docs (https://github.com/RasaHQ/rasa/blob/e0caa6377762f77258bdeadfd10492d6792a06a3/docs/docs/nlg.mdx#L31, there might be more occurrences) ?

Reply:
Updated. I'll do a pass on the docs too, good call.